### ISSUE TITLE
User interface shifts when editing comment

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#3565]
Running Mac/Chrome using today's posted build and running R/C test matrix.

Very weird thing where if I edit a comment, and then click into (an empty) description field, the majority of the Maqetta UI jumps down and to the right.

![Shifted UI](http://i.imgur.com/e3tTX.png)

Then if you pretty much do anything to the screen, the UI rectifies itself.

Wonder if it's a Chrome bug? Looking at "Elements" view in debugger, the debugger thinks that the Maqetta UI is in the right place (versus shifted as it appears on the screen).

Can't reproduce in Maqetta.org using the same version of Chrome.

Can't reproduce with today's build using Firefox.

**Comment by jhpedemonte** [comment]
Yes, I have seen this too.  Saw it when we were finishing up M7.  It went away, then I saw it again yesterday.  Seems to only happen in Chrome.  Not sure how we can even debug this.

**Comment by JonFerraiolo** [comment]
One idea is to add code that will trigger a resize event for those circumstances where the UI shifts, perhaps either via createEvent() or fussing with width/height on particular elements to trigger Chrome's layout logic.

Have you seen it in cases other than adding review comments?

**Comment by jhpedemonte** [comment]
If I remember correctly, it usually happened when I was scrolling in a view (mouse held down on scrollbar, scrolling downward).  I'd have to try again to see if I can recreate.

**Comment by JonFerraiolo** [comment]
Couldn't reproduce just now when running Mac/Chrome from Eclipse.

**Comment by JonFerraiolo** [comment]
I saw this again when doing SmartInput on a mobile widget (undoubtedly a RoundRectList because that's what I was testing).

Pushing to M9.

**Comment by JonFerraiolo** [comment]
From M10 -> future
